release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1459,6 +1459,8 @@ In the following tables, features are marked with the following statuses:
 
 * For a bonding network interface that holds a `br-ex` bridge device, do not set the `mode=6 balance-alb` bond mode in a node network configuration. This bond mode is not supported on {product-title} and it can cause the Open vSwitch (OVS) bridge device to disconnect from your networking environment. (link:https://issues.redhat.com/browse/OCPBUGS-34430[*OCPBUGS-34430*]).
 
+* When a {product-title} {product-version} cluster is deployed on a single VIOS host with "vSCSI" storage on {ibm-power-name}, the CoreOS nodes with multipath enabled fail to boot at the time of installation. (link:https://issues.redhat.com/browse/OCPBUGS-32290[*OCPBUGS-32290*]).
```

Comment:
It won't ever work with single VIOS / vscsi, and is working as designed. Can we add an update to the multipath docs at https://docs.openshift.com/container-platform/4.15/post_installation_configuration/machine-configuration-tasks.html#rhcos-enabling-multipath-day-2_post-install-machine-configuration-tasks

I think we make it a warning/note.